### ISSUE TITLE
[DRAFT] 508-defect-1 ❗ Launchblocker [SCREEN READER, COGNITION]: Form titles MUST remain present while in edit mode

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#16717]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a user with cognitive considerations, I want to know which page of the form I am editing so that I can update the right information.

## Details

Upon clicking "edit" on any of the pages within the applicant information accordion, the heading element that indicates the page disappears leaving only the form inputs behind. This may cause several issues:

1. Screen reader users will struggle to navigate through the forms as heading elements are actively inserted and removed from the page. It is also not announced to the user
1. Sighted users with cognitive considerations may lost track of their place within the multiple forms nested in an accordion, particularly on mobile.
1. Missing heading elements eliminate the opportunity to make user of aria-describedby to provide unique accessible names to the several "update page" buttons on the page.

## Steps to Recreate

1. Fill out Form 28-8832 and reach the Step 2 of 2: Review Application page.
2. Open the "Application Information" accordion.
3. Click "edit" next to Applicant Information
4. Verify the heading "Application Information" has been removed.

**Comment by 1Copenut** [comment]
@joshkimux I'd like to downstep this one to a severity 2 from a 1. There's room for improvement as you've outlined, but the critical path functionality is working. LMK if you'd like to chat more about this.

**Comment by joshkimux** [comment]
@1Copenut I've updated it accordingly! ✌️

**Comment by jason-gcio** [comment]
We see that this would be an improvement but we are not able to adjust these at this time since this is a Form System issue.  Leaving in Icebox for now.
